### PR TITLE
Adds the disabled mech tools.

### PR DESCRIPTION
**Pull request by Belsima**
Mecha jet-packs and defense shockers.

### CODE REVIEW

**Comment by Belsima** [comment]
first time using tortoise whatd i do wrong

File: code/game/mecha/equipment/tools/tools.dm [hunk 1]
```diff
@@ -1186,3 +1187,99 @@
...
+	name = "ion jetpack"
+	desc = "Using directed ion bursts and cunning solar wind reflection technique, this device enables controlled space flight."
+	icon_state = "mecha_jetpack"
+	equip_cooldown = 5
+	energy_drain = 50
+	var/wait = 0
+	var/datum/effect/effect/system/ion_trail_follow/ion_trail
+
+
+	/obj/item/mecha_parts/mecha_equipment/tool/jetpack/can_attach(obj/mecha/M as obj)
```
Comment (MisterLayne) [inline code/game/mecha/equipment/tools/tools.dm]: Everything here and down.. Is indented when it should not be.

Comment (Belsima) [reply code/game/mecha/equipment/tools/tools.dm]: dammnit

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 2]
```diff
@@ -335,3 +330,35 @@
...
+	desc = "A device to electrify the external portions of a mecha in order to increase its defensive capabilities."
+	icon_state = "mecha_coil"
+	equip_cooldown = 10
+	energy_drain = 100
+	range = RANGED
+	origin_tech = list(TECH_COMBAT = 3, TECH_POWER = 6)
+	var/shock_damage = 15
+	var/active
+
+	/obj/item/mecha_parts/mecha_equipment/shocker/can_attach(obj/mecha/M as obj)
```
Comment (MisterLayne) [inline code/game/mecha/equipment/weapons/weapons.dm]: Same problem with indentation here.

**Comment by MisterLayne** [comment]
is it sad I looked the code up and down like ten times before noticing that because I did

**Comment by ElectricB1ue** [comment]
>re-adding the clown mech honker
o h   n o

**Comment by Belsima** [comment]
@ElectricB1ue Still commented out.

**Comment by Belsima** [comment]
@Anewbe If staff say the fixed version is okay I'll uncomment it, relevant to Electric's comment.

**Pull request merged**